contracts/bridge/BridgeTokens.sol

```diff
@@ -0,0 +1,103 @@
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.5.6;
+
+import "../externals/openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract BridgeTokens is Ownable {
+    mapping(address => address) public allowedTokens; // <token, counterpart token>
+    address[] public allowedTokenList;
+    mapping(address => bool) public lockedTokens;
+
+    event TokenRegistered(address indexed token);
+    event TokenDeregistered(address indexed token);
+    event TokenLocked(address indexed token);
+    event TokenUnlocked(address indexed token);
+
+    modifier lockedToken(address _token) {
+        require(allowedTokens[_token] != address(0), "invalid token");
+        require(lockedTokens[_token] == true, "unlocked token");
+        _;
+    }
+
+    modifier unlockedToken(address _token) {
+        require(allowedTokens[_token] != address(0), "invalid token");
+        require(lockedTokens[_token] == false, "locked token");
+        _;
+    }
+
+    function getAllowedTokenList() external view returns(address[] memory) {
+        return allowedTokenList;
+    }
+
+    // registerToken can update the allowed token with the counterpart token.
+    function registerToken(address _token, address _cToken)
+        external
+        onlyOwner
+    {
+        require(allowedTokens[_token] == address(0));
+        allowedTokens[_token] = _cToken;
+        allowedTokenList.push(_token);
+
+        emit TokenRegistered(_token);
+    }
+
+    // deregisterToken can remove the token in allowedToken list.
+    function deregisterToken(address _token)
+        external
+        onlyOwner
+    {
+        require(allowedTokens[_token] != address(0));
+        delete allowedTokens[_token];
+        delete lockedTokens[_token];
+
+        for (uint i = 0; i < allowedTokenList.length; i++) {
+            if (allowedTokenList[i] == _token) {
+                allowedTokenList[i] = allowedTokenList[allowedTokenList.length-1];
+                allowedTokenList.length--;
+                break;
+            }
+        }
+
+        emit TokenDeregistered(_token);
+    }
+
+    // lockToken can lock the token to prevent request token transferring.
+    function lockToken(address _token)
+        external
+        onlyOwner
+        unlockedToken(_token)
+    {
+        lockedTokens[_token] = true;
+
+        emit TokenLocked(_token);
+    }
+
+    // unlockToken can unlock the token to  request token transferring.
+    function unlockToken(address _token)
+        external
+        onlyOwner
+        lockedToken(_token)
+    {
+        require(allowedTokens[_token] != address(0));
```

Comment:
It is already checked in the modifier `lockedToken`. 
By the way, I cannot sure we should check `allowedTokens` inside of the modifier.